README.md

```diff
@@ -53,7 +55,7 @@ This has the latest features, but is much more unstable, use at your own risk.
 
 #### Unstable Master Branch Build (with custom autopilot)
 
-This version is the same as the regular master/development version, but with the WIP custom fly-by-wire, autopilot and autothrust system. Expect issues with autopilot, engines, autothrust and the PFD if you intend to use this version. No support will be provided via Discord.
+This version is the same as the regular master/development version, but with the W.I.P. custom fly-by-wire, autopilot and autothrust system. Expect issues with autopilot, engines, autothrust and the PFD if you intend to use this version. No support will be provided via Discord.

```

Comment:
I think we could find a better adjective here. WIP is clunky, no matter whether you add the dots or not.

Reply:
How about experimental?

Reply:
Or just `work in progress`...?